public/apps/account/password-reset-panel.tsx

```diff
@@ -32,6 +32,7 @@ import { logout, updateNewPassword } from './utils';
 import { PASSWORD_INSTRUCTION } from '../apps-constants';
```

Comment:
For this `PASSWORD_INSTRUCTION`, should we also update it so that it has the new password rules that introduced by the strong password validation feature?

Reply:
Just like this `password_validation_error_message` :
```
'Password must be minimum 5 characters long and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character.'
```

Reply:
That's a good point and needs to be updated on the backend as well. I'll adjust it according to the description on the PR that introduced password strength: https://github.com/opensearch-project/security/pull/2557

> plugins.security.restapi.password_min_length - minimum password length, default and minimum is 8

Reply:
This is already properly set: https://github.com/opensearch-project/security-dashboards-plugin/blob/main/public/apps/apps-constants.tsx 

Note: This default is duplicated and the [frontend](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/public/apps/apps-constants.tsx ) and [backend](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/rest/DashboardsInfoAction.java#L68-L69). I decided to keep the value in the frontend, but it could certainly be removed from the frontend codebase in favor of just keeping a single reference on the backend.

Reply:
Let's do this as a follow up PR to unblock your change.